themes/freemind/source/css/style.css

```
@@ -1,4 +1,4 @@
-@import url('https://fonts.googleapis.com/css2?family=IBM+Plex+Mono&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600&display=swap');
```

Stylelint (17.3.0)

[error] Expected "url('https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600&display=swap')" to be "'https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600&display=swap'" (import-notation)

(import-notation)